crates/lang/ir/src/ir/item_impl/callable.rs

```diff
@@ -251,7 +251,7 @@ pub trait Callable {
 /// }
 /// ```
 ///
-/// ... then the selector of `my_message` is composed such as:
+/// …then the selector of `my_message` is composed such as:
```

Comment:
somehow I find those unicode triple dots in some positions extremely ugly and not at all an improvement. is it just me?